src/utils/power_flow.jl

```diff
@@ -126,7 +126,7 @@ Updates system voltages and powers with power flow results
 """
 function _write_pf_sol!(sys::System, nl_result)
     result = round.(nl_result.zero; digits = 7)
-    buses = enumerate(sort(collect(get_components(Bus, sys)), by = x -> get_number(x)))
+    buses = enumerate(sort!(collect(get_components(Bus, sys)), by = x -> get_number(x)))
```

Comment:
The linter did not actually suggest this change.  In general, I think that we should sort in place after collect rather than allocate a new array.   I understand that some of these changes are in performance-sensitive areas and if you prefer that I not make them, I can remove.

Reply:
I think it makes sense to do this but then it will require collecting first and then running `sort!()`.